widgetry/src/text.rs

```diff
@@ -540,16 +540,26 @@ impl TextSpan {
         assets: &A,
         path: &PolyLine,
         scale: f64,
+        color: Color,
     ) -> GeomBatch {
         let assets = assets.as_ref();
         let tolerance = svg::HIGH_QUALITY;
 
         // Just set a sufficiently large view box
         let mut svg = r##"<svg width="9999" height="9999" viewBox="0 0 9999 9999" xmlns="http://www.w3.org/2000/svg">"##.to_string();
-
+        let height = (Text::from(&self.text).dims(assets).height * scale);
+        let width = (Text::from(&self.text).dims(assets).width * scale);
+        write!(
+            &mut svg,
+            r##"<rect x="0" y="0" width="{}" height="{}" fill="{}" />"##,
```

Comment:
I don't think this approach works. :(

It adds an axis-oriented (no rotation) rectangle based on the text's width & height if we drew it oriented normally, and placed at 0,0. But then below we draw the text translated and oriented to follow a curve. I changed the background color to cyan just to understand what's going on, and didn't see the rectangle under the labels. Instead it's at the top-left corner of the map:

![Screenshot from 2022-04-10 21-40-36](https://user-images.githubusercontent.com/1664407/162638872-6b72974b-eaaa-4127-9293-b69f25c2b34f.png)

I started playing with other ideas to deal with this, like setting `fill="cyan"` on the `<path id="txtpath"` part, but it doesn't do anything nice:
![Screenshot from 2022-04-10 21-41-52](https://user-images.githubusercontent.com/1664407/162638954-d57abed9-8ecd-41e0-a962-003f274a94c6.png)
I wouldn't expect it to either...

Thinking about this problem a bit more, sometimes the label straddles two lanes, which might be different colors, like this:
![Screenshot from 2022-04-10 21-44-48](https://user-images.githubusercontent.com/1664407/162639012-37252f03-6bcd-4f99-8310-50a176f4992f.png)
Even if we calculated a polygon that follows the curved text, picking one of the lane's colors and putting a background wouldn't really work. Really what we want to do here is...

1) Not draw the dashed center line underneath the label
2) Rethink the color of the label, so it'll still show up reasonably clear on these different lane colors

Not sure how to do either easily. :\

Reply:
Looking at the pics, a black label might work? Alternately, what I did with my space game where I had all sorts of planet/nebula colors was to draw a background behind it (gray) and then black text. This ensured the labels are always readable regardless of what crazy color it happened to be on.

Reply:
Hey guys,

Thanks for the review :). 

1) I will do this
2) Does SVG allow a stroke color? We could do black stroke with white text